### DISCUSSION TITLE
Welcome to Tinker9 Discussions!

### DISCUSSION THREAD

**Discussion by zhi-wang** [TinkerTools/tinker-gpu#111] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by asedova** [comment]
Hi all! A question about the gcc 4.8 warning. It seems that 4.8.5 should be c++11 complete (4.8.0 was not). Have you found anything that indicates this is not the case?

**Reply by zhi-wang** [reply]
I had some difficulties to look for workarounds with 4.8.5 in the early stage of this code base. I think I have resolved the problems I could find but I decided to keep this warning in the code. I want to discourage people to use this ancient compiler.

One of the problem was that the regular expression engine, regex, which is part of C++11 standard was an EMPTY implementation. I could not use it at all.

Reference:
https://stackoverflow.com/questions/12530406

**Reply by asedova** [reply]
ok... we have the same issue with the pgi being built with gcc 4.8.5. the local config is a little tricky, and I wanted something to complain about to the admins so they would rebuild pgi with a different config, like a specific issue from the code.

**Reply by zhi-wang** [reply]
I believe you have read about the tricks to re-do the local config in the documentation: https://github.com/TinkerTools/tinker9/blob/master/doc/manual/m/install/preq.rst

I wish this trick will work in your IBM environment as well before the admins are able to make the global change. And I think having a "real" standard-compliant C++11 compiler is probably a very reasonable request nowadays.
